utils/ecl-diff/ecl-diff-component.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, no-param-reassign */
+
+const prettier = require('prettier');
+const fs = require('fs');
+const eclDiffVariant = require('./ecl-diff-variant.js');
+
+const rootFolder = process.cwd();
+const distFolder = `${rootFolder}/php`;
+// We retrieve all the available variants files (php ones).
+const getData = (component, system) => {
+  const systemFolder = `${distFolder}/packages/${system}`;
+  const twigFullPath = `${systemFolder}/${component}`;
+  const twigFilesFolder = fs.readdirSync(`${twigFullPath}`);
+  const twigFiles = twigFilesFolder.filter((elm) => {
+    return elm.match(/.*\.(php.html)/gi);
+  });
+
+  const variants = [];
+  twigFiles.forEach(async (twigFile) => {
```

Comment:
Why the `async` here? I don't see an `await` inside or a promise. You gotta be careful with `async/await` and `forEach` https://www.coreycleary.me/why-does-async-await-in-a-foreach-not-actually-await/